src/core/observables/FluxDensityProfile.hpp

```diff
@@ -19,9 +19,9 @@ class FluxDensityProfile : public ProfileObservable {
     Utils::Histogram<double> histogram(n_bins, 3, limits);
     for (int id : ids) {
       auto const ppos = ::Vector<3, double>(folded_position(partCfg[id]));
-      histogram.update(ppos, ::Vector<3, double> {{partCfg[id].m.v[0],
-                                                   partCfg[id].m.v[1],
-                                                   partCfg[id].m.v[2]}});
+      histogram.update(ppos, ::Vector<3, double> {{partCfg[id].m.v[0] / time_step,
+                                                   partCfg[id].m.v[1] / time_step,
+                                                   partCfg[id].m.v[2] / time_step}});
```

Comment:
Are you sure they should be divided by the time step and not multiplied? The units on these weights look a bit odd.

Reply:
Actually this is correct and tested in observable_profile.